packages/react/src/App.tsx

```diff
@@ -36,6 +37,8 @@ function RootApp() {
 export const App = () => {
   const screenSizeClass = useScreenSizeClass();
 
+  switchTheme();
```

Comment:
Do we really need to call switchTheme here? It will be called anyway on ThemeSwitcher mount from useEffectHook.

Reply:
right, it works without calling switchTheme() here